### ISSUE TITLE
Bug hits contact magia

### ISSUE THREAD

**Issue by Stevemalboro** [ikemen-engine/Ikemen-GO#1533]
### Describe the bug

This bug consists of the following: When two characters release powers at the same time and the spells collide with each other, the hit counter counts as if the character had been touched. Version 0.98.2 does not have this bug, but the most recent version 0.99 .0 came with this b
![bu](https://github.com/ikemen-engine/Ikemen-GO/assets/152203259/5744919d-6a31-4615-95ca-273c434fbfa6)
ug.

### To Reproduce

![bu](https://github.com/ikemen-engine/Ikemen-GO/assets/152203259/079b934c-4bc7-4d45-83de-461367b6338d)
As you can see here the spells collide and give hits as if they were touching the char's collision box, this in version 099.0.2

### Expected behavior

![bu2](https://github.com/ikemen-engine/Ikemen-GO/assets/152203259/2d22b5d9-375c-4b86-b99f-6f7a78782f70)
Already in version 098.2. this doesn `t happen .

### Screenshots / Video

https://github.com/ikemen-engine/Ikemen-GO/assets/152203259/f9d1ad0e-45a7-45fb-b211-6ba57783a7fc

### Engine Version (or source code date)

v099.0.2

### Operating system

Windows

### Extra context or search terms

Well, exactly that, it happens like this when magic collides with each other and a large sequence ends up counting hits.

**Comment by potsmugen** [comment]
Sounds like same issue fixed by #1515
Did you try it on the Nightly build?

**Comment by Stevemalboro** [comment]
> Parece o mesmo problema corrigido por [# 1515.](https://github.com/ikemen-engine/Ikemen-GO/pull/1515) Você tentou na versão Nightly?

No, I just downloaded the latest version from github, which is there

**Comment by potsmugen** [comment]
Try with this one:
https://github.com/ikemen-engine/Ikemen-GO/releases/tag/nightly

Alternatively upload a character with that bug.

**Comment by Stevemalboro** [comment]
> Experimente este: https://github.com/ikemen-engine/Ikemen-GO/releases/tag/nightly
> 
> Alternativamente, carregue um personagem com esse bug.

Perfect my friend, there are no bugs in this one, what would this version be? This one is good. Thank you very much my friend without words, Ikemen forever, gtihub forever.

**Comment by Stevemalboro** [comment]
> > Experimente este: https://github.com/ikemen-engine/Ikemen-GO/releases/tag/nightly
> > Alternativamente, carregue um personagem com esse bug.
> 
> Perfeito meu amigo, não há bugs neste, qual seria essa versão? Este é bom. Muito obrigado meu amigo sem palavras, Ikemen para sempre, gtihub para sempre.



> Experimente este: https://github.com/ikemen-engine/Ikemen-GO/releases/tag/nightly
> 
> Alternativamente, carregue um personagem com esse bug.

NO more Bugs Thanks my friend
